apps/client/src/pages/ConclusionPage.js

```diff
@@ -99,6 +99,68 @@ const ConclusionPage = ({ topic, checker }) => {
   };
   const currentDateTime = date.toLocaleDateString("nl-NL", options);
 
+  // Demo saveConclusion function
+  const saveConclusion = () => {
+    // Demo getRegistryAnswer
+    const getRegistryAnswer = (id) => {
+      // How do we get the registry answer?
+      return "true";
+    };
+
+    // Demo getQuestionAndAnswers
+    const getQuestionAndAnswers = () => {
+      return this.stack.reduce((acc, question) => {
+        acc.push({
+          ...question,
+          // id: does `question` contain `id`? (Yes)
+          // text: does `question` contain `text`?
+          // answer: does `question` contain `answer`? // key: `answer` or `userAnswer`? (Yes)
```

Comment:
Yes, we have `answer`. I think that's a good name for it too.
Please note the value can be a list of answers (`string[]`).

Reply:
Fixed in another way